Character creation guidelines

djigneo opened the pull request:
Updated guidelines to remove language around Session 0 and DMs.

djigneo commented:
Sorry for the file conflicts. My Git skills apparently aren't that strong.

I'm not sure what to say about the Issue template. Perhaps it would be helpful to separate the mandatory checklist steps from the "helpful to think about" steps, with the mandatory steps going into the Issue template?

The way I see it, the "step-by-step characters" such as it was are guidelines for helping a player think through the character, but they are really _optional_ steps to making a character that's playable in this game. It's a supplement, not a requirement, so perhaps the checklist which appears in the Issue template are all the absolutely mandatory parts.

To that end, it perhaps makes sense to split out the character creation parts that are more like "rules" into a separate document as well to make an explicit distinction between the guidelines for the game and supplemental information to help with character creation.

zmoazeni commented:
Yeah I'm totally with you. I like the idea of us splitting that way:

1. The stuff in the issue template mentions the things that are mandatory while hinting at the things encouraged. It also links to other areas in the repository for more information.
2. I do really like the vast majority of the content you posted here. I'll go through it a little more thoroughly later, but I feel they compliment each other well. The README will link to this, and perhaps so will different sections of the issue template to help guide the person.

I really just want to avoid someone having to read _everything_ just to join a game. If they can start by creating an issue template and it communicates everything they need to do, that sounds like it'll be swell.

For the record, I want to do the exact same thing with the session template. I need to do a better job of that guiding people on the easiest path to run a game too.

djigneo commented:
Reorganized content:
- Modified Character issue template to be a simplified checklist with links to other resources.
- Split out guidelines / rules around character creation from in-depth character-building content
- Created separate *md file for wiki template page

Hopefully this gets us a little closer to organized. Welcome comments on updated organization.

crimsonknave commented on .github/ISSUE_TEMPLATE/character.md:
```diff
@@ -6,46 +6,9 @@ labels: 'character,needs-approval'
-_People are flawed. Not even you dear adventurer are perfect! Has your character pissed someone off? Are they outlawed from a certain town for a... misunderstanding? Do they struggle to overcome something: a paranoia, a physical limitation, an addiction?_ 
-
-_This is entirely optional, but depending on the flare the approving GM may award an XP bonus while your character struggles with their demons._
+[ ] Review the [character creation guidelines](character%20creation%20guidelines.md)
```
```suggestion
- [ ] Review the [character creation guidelines](character%20creation%20guidelines.md)
```

crimsonknave commented on .github/ISSUE_TEMPLATE/character.md:
```diff
@@ -6,46 +6,9 @@ labels: 'character,needs-approval'
-
-_This is entirely optional, but depending on the flare the approving GM may award an XP bonus while your character struggles with their demons._
+[ ] Review the [character creation guidelines](character%20creation%20guidelines.md)
+[ ] [Make a Wiki page](character%20wiki%20page%20outline.md) for your character <fill in link here>
```
```suggestion
- [ ] [Make a Wiki page](character%20wiki%20page%20outline.md) for your character <fill in link here>
```

crimsonknave commented on .github/ISSUE_TEMPLATE/character.md:
```diff
@@ -6,46 +6,9 @@ labels: 'character,needs-approval'
-_This is entirely optional, but depending on the flare the approving GM may award an XP bonus while your character struggles with their demons._
+[ ] Review the [character creation guidelines](character%20creation%20guidelines.md)
+[ ] [Make a Wiki page](character%20wiki%20page%20outline.md) for your character <fill in link here>
+[ ] Make a character sheet on [DndBeyond](www.dndbeyond.com) [fill in link here]
```
```suggestion
- [ ] Make a character sheet on [DndBeyond](www.dndbeyond.com) [fill in link here]
```

crimsonknave commented on .github/ISSUE_TEMPLATE/character.md:
```diff
@@ -6,46 +6,9 @@ labels: 'character,needs-approval'
+[ ] Review the [character creation guidelines](character%20creation%20guidelines.md)
+[ ] [Make a Wiki page](character%20wiki%20page%20outline.md) for your character <fill in link here>
+[ ] Make a character sheet on [DndBeyond](www.dndbeyond.com) [fill in link here]
+[ ] Develop a character concept further - See [Step by Step Characters](step%20%20by%20characters.md)
```
```suggestion
- [ ] Develop a character concept further - See [Step by Step Characters](step%20%20by%20characters.md)
```

crimsonknave commented on .github/ISSUE_TEMPLATE/character.md:
```diff
@@ -6,46 +6,9 @@ labels: 'character,needs-approval'
+[ ] [Make a Wiki page](character%20wiki%20page%20outline.md) for your character <fill in link here>
+[ ] Make a character sheet on [DndBeyond](www.dndbeyond.com) [fill in link here]
+[ ] Develop a character concept further - See [Step by Step Characters](step%20%20by%20characters.md)
+[ ] Create connections and interact with other characters / players in the community within this issue
```
```suggestion
- [ ] Create connections and interact with other characters / players in the community within this issue
```

crimsonknave commented on .github/ISSUE_TEMPLATE/character.md:
```diff
@@ -6,46 +6,9 @@ labels: 'character,needs-approval'
+[ ] Make a character sheet on [DndBeyond](www.dndbeyond.com) [fill in link here]
+[ ] Develop a character concept further - See [Step by Step Characters](step%20%20by%20characters.md)
+[ ] Create connections and interact with other characters / players in the community within this issue
+[ ] Join or organize a Session
```
```suggestion
- [ ] Join or organize a Session
```

crimsonknave commented on character creation guidelines.md:
```diff
@@ -0,0 +1,16 @@
+
+## Character level
+You can create a character at any level, but it is at the GMs discretion to run games for the character levels they feel comfortable with.
+General guideline is to create a character between levels 1 and 3.
```
I think this should note that as the party increases in level it is also fine to make a character of a level that others in the campaign are at.

djigneo replied:
Done.

crimsonknave commented on step by step characters.md:
```diff
@@ -0,0 +1,64 @@
+# Step-by-step characters
```
Isn't everything in this file included in the file above already?

djigneo replied:
No, if you take a look at this branch, _step by step characters_ was split from _character creation guidelines_ into its own file.

crimsonknave replied:
I'm confused. That's the branch this PR is based on, right? The character-creation.md file that's right above still has the content.
